sdk/formrecognizer/ai-form-recognizer/tests.yml

```
@@ -15,8 +15,21 @@ extends:
     parameters:
       PackageName: "@azure/ai-form-recognizer"
       ServiceDirectory: formrecognizer
-      Location: "${{ parameters.Location }}"
-      EnvVars:
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+      TimeoutInMinutes: 120
+      SupportedClouds: "Public,UsGov,China"
+      CloudConfig:
+        Public:
+          ServiceConnection: azure-sdk-tests
+          SubscriptionConfigurationFilePaths:
+            - eng/common/TestResources/sub-config/AzurePublicMsft.json
+          Location: "${{ parameters.Location }}"
+        UsGov:
+          ServiceConnection: usgov_azure-sdk-tests
+          SubscriptionConfigurationFilePaths:
+            - eng/common/TestResources/sub-config/AzureUsGovMsft.json
+          Location: "usgovvirginia"
+        China:
+          ServiceConnection: china_azure-sdk-tests
+          SubscriptionConfigurationFilePaths:
+            - eng/common/TestResources/sub-config/AzureChinaMsft.json
+          Location: "chinaeast2"
```

```suggestion
      TimeoutInMinutes: 120
      SupportedClouds: "Public,UsGov,China"
      CloudConfig:
        Public:
          Location: "${{ parameters.Location }}"
        UsGov:
        China:
          Location: "chinaeast2"
```

We've simplified to not have to define redundant information anymore.